.github/workflows/ci.yml

```
@@ -49,19 +49,16 @@ jobs:
         rust: [stable]
         python-version:
           [
-            "3.7",
-            "3.8",
             "3.9",
             "3.10",
-            "3.11-dev",
-            "pypy-3.7",
-            "pypy-3.8",
+            "3.11",
+            "3.12",
             "pypy-3.9",
           ]
         platform:
           [
             {
-              os: "macOS-latest",
+              os: "macos-latest",
               python-architecture: "x64",
               rust-target: "x86_64-apple-darwin",
```

`macos-latest` runners are now arm:

```suggestion
              python-architecture: "arm64",
              rust-target: "aarch64-apple-darwin",
```